src/spell-kind/spells-sight.cpp

```diff
@@ -461,22 +461,26 @@ bool probing(PlayerType *player_ptr)
         }
         msg_print(nullptr);
 
-        probed_monster_info(buf, player_ptr, m_ptr, r_ptr);
-        prt(buf, 0, 0);
+        const auto probe_result = probed_monster_info(player_ptr, m_ptr, r_ptr);
+        prt(probe_result.data(), 0, 0);
 
-        message_add(buf);
+        message_add(probe_result);
         player_ptr->window_flags |= (PW_MESSAGE);
         handle_stuff(player_ptr);
         move_cursor_relative(m_ptr->fy, m_ptr->fx);
         inkey();
         term_erase(0, 0, 255);
         if (lore_do_probe(player_ptr, m_ptr->r_idx)) {
-            strcpy(buf, (r_ptr->name.data()));
 #ifdef JP
-            msg_format("%sについてさらに詳しくなった気がする。", buf);
+            msg_format("%sについてさらに詳しくなった気がする。", r_ptr->name.data());
 #else
-            plural_aux(buf);
-            msg_format("You now know more about %s.", buf);
+            {
```

Comment:
This curly brackets is no longer needed.
(I guess buf is duplicated)

Reply:
The latest forced push removed the extra nesting of braces for the section setting up the English message.